.github/workflows/build_packages.yaml

```diff
@@ -164,6 +164,31 @@ jobs:
         kerl update releases
         kerl build ${{ matrix.otp }}
         kerl install ${{ matrix.otp }} $HOME/.kerl/${{ matrix.otp }}
+
+    - name: Get deps git refs for cache
+      id: deps-refs
+      run: |
```

Comment:
Seems like this is repeated multiple times. Move it to a script?

Reply:
Another possibility would be to maybe use the new [composite actions](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action).